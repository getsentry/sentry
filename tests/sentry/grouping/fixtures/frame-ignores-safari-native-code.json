{
  "stacktrace": {
    "frames": [
      {
        "function": "forEach",
        "abs_path": "[native code]",
        "filename": "[native code]"
      }
    ]
  }
}
