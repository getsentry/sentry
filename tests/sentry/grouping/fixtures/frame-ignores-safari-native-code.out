app:
  hash: None
  component:
    app
      stacktrace
        frame (frame considered in-app because no frame is in-app)
          function (function name is used only if module or filename are available)
            u'forEach'
--------------------------------------------------------------------------
fallback:
  hash: 'd41d8cd98f00b204e9800998ecf8427e'
--------------------------------------------------------------------------
system:
  hash: None
  component:
    system
      stacktrace
        frame (native code indicated by filename)
          function (function name is used only if module or filename are available)
            u'forEach'
